src/app/components/commons/Header.tsx

```diff
@@ -1,34 +1,38 @@
 import * as React from "react";
-import {AppBar, Button, createStyles, IconButton, SvgIcon, Theme, Toolbar} from "@material-ui/core";
+import {AppBar, Button, createStyles, IconButton, SvgIcon, Theme, Toolbar, Typography} from "@material-ui/core";
 import {makeStyles} from "@material-ui/core/styles";
 import HeaderIcon from "../../icons/header-icon.svg";
 
 const useStyles = makeStyles((theme: Theme) =>
     createStyles({
-        root: {
-            flexGrow: 1,
+        typography: {
+            flexGrow: 0.85,
+            textAlign: 'center'
         },
         menuButton: {
             marginRight: theme.spacing(2),
         },
         appBar: {
-            marginBottom: '10px'
-        }
+            marginBottom: '10px',
+        },
     }),
 );
 
 export const Header = () => {
     const classes = useStyles();
 
     return (
-        <AppBar position='static' className={classes.appBar}>
-            <Toolbar>
-                <IconButton edge="start" className={classes.menuButton} color="inherit" aria-label="menu">
-                    <SvgIcon component={HeaderIcon} viewBox="0 0 500 500" color="secondary"/>
-                </IconButton>
-                <Button href="/" color="inherit">Home</Button>
-                <Button href="/champion" color="inherit">Champions</Button>
-            </Toolbar>
-        </AppBar>
+        <React.Fragment>
+            <AppBar position='static' className={classes.appBar}>
+                <Toolbar>
+                    <IconButton edge="start" className={classes.menuButton} color="inherit" aria-label="menu">
+                        <SvgIcon component={HeaderIcon} viewBox="0 0 500 500" color="secondary"/>
+                    </IconButton>
+                    <Button href="/" color="inherit">Home</Button>
+                    <Button href="/champion" color="inherit">Champions</Button>
+                    <Typography variant="h4" className={classes.typography}>Stat Check</Typography>
```

Comment:
I'm not a fan of those typography. It make sense to me that you have `<h2>` with the right size defined in your GlobalStyleProvider.
And please, respect the h1/h2/h3/h4 hierarchy (no h4 without h3 in your page). Do not use them in a way that looks good. Adapt the size for the whole application on the way you will use them.